apel_rest/urls.py

```diff
@@ -12,5 +12,7 @@
 
     url(r'^admin/', include(admin.site.urls)),
     url(r'^api/v1/cloud/record$', CloudRecordView.as_view()),
-    url(r'^api/v1/cloud/record/summary$', CloudRecordSummaryView.as_view())
+    url(r'^api/v1/cloud/record/summary$', CloudRecordSummaryView.as_view()),
+    url(r'^accounting-server/api/v1/cloud/record$', CloudRecordView.as_view()),
```

Comment:
What is the reason for duplicating the API under different parent branches?

Could this be a config option?